docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: '3'
```

Comment:
If this is just for the dev db, does it need to be in the PR?

Reply:
We could also use this for tests as well. Might make sense to commit with a sample test so that it can be verified as functioning?